lisp/lisplib/libschema/README.md

```diff
@@ -16,26 +16,20 @@ Types are defined using the `s:deftype` keyword and validations are performed by
 We can validate that a value meets the required type by calling `s:validate` on it with the required value:
 ```lisp
 (set 'x "hello")
-(assert-nil (s:validate s:string x))
+(s:deftype "mystring" s:string)
+(assert-nil (s:validate mystring x))
 ```
 
 If the value does not have the required type, an error of type `"wrong-type"` will be returned. If a constraint (see below)
 fails, an error of type `"failed-constraint"` will be the result.
 
-The type passed to `s:validate` con be either a primitive type validator
-function or it can be a lisp:typespec created by by the `lisp:deftype` macro,
-in which case `s:validate` checks the typedef matches the type of the given
-value and then run any type validation that was defined with `s:deftype`.
+If using `s:make-validator` to define a time then pass the validator it
+returned along weth the value to be validated.
```

Comment:
`weth` is this a typo?

Reply:
At this point I think I wrote too many docs for this feature and just gave up.